tofu/geom/_comp_solidangles.py

```diff
@@ -196,7 +196,7 @@ def calc_solidangle_particle(
     # Main computation
 
     # traj2pts vector, with length (3d array (3, N, M))
-    vect = traj[:, :, None] - pts[:, None, :]
+    vect = pts[:, :, None] - traj[:, None, :]
```

Comment:
Why ?
The unit vector to be returned is the unit vector giving the direction from the plasma pts to the particle trajectory:
vect = traj - pts

Because it gives the direction of propagation of the photons emitted by the plasma towards the particle.